README.md

```diff
@@ -92,41 +92,53 @@ end program simple_get
-
-
-## **Contributing to project**
+## **Getting Started Guides** 👋
+
+
+### **Table of contents:** 
+
+1. ### [**Installation** 🌈](tutorial/tuotrial.md#installation)
+
+2. ### [**Making HTTP Requests** 🚀](tutorial/tuotrial.md#making-http-requests-f09f9a80-1)
+    - [**Sending `GET` Requests**](tutorial/tuotrial.md#sending-get-requests)
+        - [*Accessing Response `Content`*](tutorial/tuotrial.md#accessing-response-content)
+        - [*Extracting `Content Length`*](tutorial/tuotrial.md#extracting-content-length)
+        - [*Retrieving `Status Codes`*](tutorial/tuotrial.md#retrieving-status-codes)
+        - [*Handling `Errors`*](tutorial/tuotrial.md#handling-errors)
+        - [*Getting Response `Headers`*](tutorial/tuotrial.md#getting-response-headers)
+        - [*Understanding `pair_type` derived type*](tutorial/tuotrial.md#understanding-pair_type-derived-type)
+        - [*Sending Custom `Headers`*](tutorial/tuotrial.md#sending-custom-headers)
+        - [*Setting Request `Timeout`*](tutorial/tuotrial.md#setting-request-timeout)
+        - [*Setting `Authentication`*](tutorial/tuotrial.md#setting-authentication)
+    - [**Sending `POST` Requests**](tutorial/tuotrial.md#sending-post-request)
+        - [*Sending Data using `data`*](tutorial/tuotrial.md#sending-data-using-data)
+        - [*Sending Data using `form`*](tutorial/tuotrial.md#sending-data-using-form)
+        - [*Sending Data using `file`*](tutorial/tuotrial.md#sending-data-using-file)
+    - [**Sending `PUT` Requests**](tutorial/tuotrial.md#sending-put-requests)
+    - [**Sending `PATCH` Requests**](tutorial/tuotrial.md#sending-patch-requests)
+    - [**Sending `DELETE` Requests**](tutorial/tuotrial.md#sending-delete-requests)
+    - [**Sending `HEAD` Requests**](tutorial/tuotrial.md#sending-head-requests)
+
+3. ### [**Real Projects** 🤖](tutorial/tuotrial.md#real-projects-f09fa496-1)
+
+## **Projects Currently using `http` package** 🥳
+
+* **[github-org-analyzer](https://github.com/rajkumardongre/github-org-analyzer):** This Fortran project **provides procedures to analyze GitHub organizations and retrieve valuable information about their repositories**. By leveraging the power of the `http-client` package, this analyzer fetches data from the GitHub API to generate insightful reports.
+
+    *~By [Rajkumar Dongre (rajkumardongre)](https://github.com/rajkumardongre)*
+* **[fortran-xkcd](https://github.com/rajkumardongre/fortran-xkcd/tree/http_client_version):** An fpm example project written in Fortran 2018 that displays the latest xkcd comic inside an X window. As a limitation, only images in PNG format are supported (no JPEG). The alt text will be printed to console.
+
+    *~By [Philipp (interkosmos)](https://github.com/interkosmos)*
+
+* **[foropenai](https://github.com/gha3mi/foropenai):** A Fortran library for OpenAI API.
+
+    *~By [AliG (gha3mi)](https://github.com/gha3mi)*
```

Comment:
What does this library?

Reply:
The primary purpose of this project is to facilitate interaction with the OpenAI API. To achieve this, the project is using `http` package for making API calls.